source/Halibut.Tests/Support/TestCases/ClientAndServiceTestCasesBuilder.cs

```diff
@@ -40,5 +45,37 @@ public IEnumerable<ClientAndServiceTestCase> Build()
                 }
             }
         }
+
+        ClientAndServiceTestVersion GetStableClientAndServiceVersionForServiceType(ServiceConnectionType serviceConnectionType, Version? clientVersion, Version? serviceVersion)
```

Comment:
This gives us an extension point if we find similar issues with other service versions